### PR TITLE
ref(alerts): Add docs for dynamic alerts

### PR DESCRIPTION
**Pull request by ceorourke**
Add documentation for dynamic alerts. This is not urgent, probably won't be merged for a while as we'll be doing internal testing before releasing to EA.

Closes https://getsentry.atlassian.net/browse/ALRT-213

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: docs/product/alerts/create-alerts/metric-alert-config.mdx [hunk 1]
```diff
@@ -113,10 +113,11 @@ While Sentry won’t allow you to create new alerts with invalid or unavailable
 
 ## Thresholds
 
-There are two threshold types:
+There are three threshold types:
 
 - **Static**: A fixed threshold, such as when there are 100 errors in a period of time.
-- **Percent change**: A dynamic threshold, such as when there are 10% more errors in a time period compared to a previous period. These are also referred to as [Change Alerts](#change-alerts-percent-change).
+- **Percent change**: A percent based threshold, such as when there are 10% more errors in a time period compared to a previous period. These are also referred to as [Change Alerts](#change-alerts-percent-change).
```
Comment (vivianyentran) [inline docs/product/alerts/create-alerts/metric-alert-config.mdx:119]:
```suggestion
- **Percent change**: A percent-based threshold, such as when there are 10% more errors in a time period compared to a previous period. These are also referred to as [Change Alerts](#change-alerts-percent-change).
```

File: docs/product/alerts/create-alerts/metric-alert-config.mdx [hunk 2]
```diff
@@ -128,6 +129,16 @@ Change alerts, or alerts that use a percent change threshold, are useful when yo
 
 ![When the percent change option is selected.](./img/percent-change.png)
 
+### Dynamic Alerts
+
+<Include name="feature-available-for-user-group-early-adopter.mdx" />
+
+Dynamic alerts can be used when you don't know what threshold to set, but you know you want to be alerted when something is far outside the bounds of normalcy. Sentry will look at the historical data for the given metric and determine if the current data is anomalous. You can select how responsive the alert will be, and whether you want to be alerted when the metric is above, below, or both above and below the expected bounds.
```
Comment (vivianyentran) [inline docs/product/alerts/create-alerts/metric-alert-config.mdx:136]:
```suggestion
Dynamic alerts can be used when you don't know what threshold to set, but you know you want to be alerted when something is far outside the bounds of normalcy. Sentry will look at the historical data for the given metric and determine if the current data is anomalous. You can select how responsive the alert will be, and whether you want to be alerted when the metric is above and/or below the expected bounds.
```

File: docs/product/alerts/create-alerts/metric-alert-config.mdx [hunk 3]
```diff
@@ -128,6 +129,16 @@ Change alerts, or alerts that use a percent change threshold, are useful when yo
...
+
+Dynamic alerts do not use the Critical, Warning, and Resolved thresholds like other alert threshold types as you are not setting these yourself. When no more anomalies are detected, the alert will resolve.
```
Comment (vivianyentran) [inline docs/product/alerts/create-alerts/metric-alert-config.mdx:138]: I'm not sure why it matters that "you're not setting these yourself". Can you rephrase what you're trying to get across here or maybe we can just remove it so it's easier to read?

```suggestion
Dynamic alerts do not use the Critical, Warning, and Resolved thresholds. When no more anomalies are detected, the alert will resolve on its own.
```

**Review by vivianyentran — APPROVED** [review]
Thanks for preparing these docs early!

File: docs/product/alerts/create-alerts/metric-alert-config.mdx [hunk 4]
```diff
@@ -113,10 +113,11 @@ While Sentry won’t allow you to create new alerts with invalid or unavailable
 
 ## Thresholds
 
-There are two threshold types:
+There are three threshold types:
 
 - **Static**: A fixed threshold, such as when there are 100 errors in a period of time.
-- **Percent change**: A dynamic threshold, such as when there are 10% more errors in a time period compared to a previous period. These are also referred to as [Change Alerts](#change-alerts-percent-change).
+- **Percent change**: A percent-based threshold, such as when there are 10% more errors in a time period compared to a previous period. These are also referred to as [Change Alerts](#change-alerts-percent-change).
```
Comment (lizokm) [inline docs/product/alerts/create-alerts/metric-alert-config.mdx:119]:
```suggestion
- **Percent change**: A percent-based threshold set by you. (For example, if 10% more errors are detected in a time period compared to a previous period.) These are also referred to as [Change Alerts](#change-alerts-percent-change).
```

File: docs/product/alerts/alert-types.mdx [hunk 5]
```diff
@@ -37,6 +37,10 @@ The **Alert Details** page also includes a list of issues that triggered the ale
 
 Metric alerts tell you when a [metric](/product/performance/metrics/) crosses a threshold set by you, like a spike in the number of errors in a project, or a change in a performance metric, such as [transaction duration](/product/performance/metrics/#latency), [Apdex](/product/performance/metrics/#apdex), [failure rate](/product/performance/metrics/#failure-rate), or [throughput](/product/performance/metrics/#throughput-total-tpm-tps).
 
+<Note>
+You can use [dynamic alerts](/product/alerts/create-alerts/metric-alert-config/#dynamic-alerts) to let Sentry define the threshold for you.
+</Note>
+
```
Comment (lizokm) [inline docs/product/alerts/alert-types.mdx:43]:
```suggestion
You can use [dynamic alerts](/product/alerts/create-alerts/metric-alert-config/#dynamic-alerts) to let Sentry define the threshold for you.

```
I'd take this out of the note format and just add it to the paragraph above. There are a lot of different notes in this section which makes reading a bit jarring.

File: docs/product/alerts/create-alerts/metric-alert-config.mdx [hunk 6]
```diff
@@ -113,10 +113,11 @@ While Sentry won’t allow you to create new alerts with invalid or unavailable
 
 ## Thresholds
 
-There are two threshold types:
+There are three threshold types:
 
 - **Static**: A fixed threshold, such as when there are 100 errors in a period of time.
```
Comment (lizokm) [inline docs/product/alerts/create-alerts/metric-alert-config.mdx:118]:
```suggestion
- **Static**: A fixed numerical threshold set by you. (For example, if there are 100 errors in a set period of time.)
```

File: docs/product/alerts/create-alerts/metric-alert-config.mdx [hunk 7]
```diff
@@ -113,10 +113,11 @@ While Sentry won’t allow you to create new alerts with invalid or unavailable
...
-- **Percent change**: A dynamic threshold, such as when there are 10% more errors in a time period compared to a previous period. These are also referred to as [Change Alerts](#change-alerts-percent-change).
+- **Percent change**: A percent-based threshold, such as when there are 10% more errors in a time period compared to a previous period. These are also referred to as [Change Alerts](#change-alerts-percent-change).
+- **Dynamic**: A dynamic threshold set by Sentry that detects anomalies whenever values fall outside expected bounds.
```
Comment (lizokm) [inline docs/product/alerts/create-alerts/metric-alert-config.mdx:120]:
```suggestion
- **Dynamic**: A dynamic threshold set by Sentry that detects anomalies whenever values fall outside of expected bounds.
```

File: docs/product/alerts/create-alerts/metric-alert-config.mdx [hunk 8]
```diff
@@ -128,6 +129,16 @@ Change alerts, or alerts that use a percent change threshold, are useful when yo
 
 ![When the percent change option is selected.](./img/percent-change.png)
 
+### Dynamic Alerts
+
+<Include name="feature-available-for-user-group-early-adopter.mdx" />
+
+Dynamic alerts can be used when you don't know what threshold to set, but you know you want to be alerted when something is far outside the bounds of normalcy. Sentry will look at the historical data for the given metric and determine if the current data is anomalous. You can select how responsive the alert will be, and whether you want to be alerted when the metric is above and/or below the expected bounds.
+
+Dynamic alerts do not use the Critical, Warning, and Resolved thresholds. When no more anomalies are detected, the alert will resolve on its own.
```
Comment (lizokm) [inline docs/product/alerts/create-alerts/metric-alert-config.mdx:138]:
```suggestion
Dynamic alerts can be used when you're not sure about what threshold to set, but want to be alerted when something is far outside the bounds of normalcy. To determine what is and isn't normal, Sentry will look at the historical data for the given metric and compare it to the current data to see if it's anomalous. You can select how responsive the alert will be, and whether you want to be alerted when the metric is above and/or below the expected bounds.

Dynamic alerts don't use the Critical, Warning, and Resolved thresholds. When no more anomalies are detected, the alert will resolve on its own.
```

**Review by lizokm — APPROVED** [review]
Thanks for adding this! I left a few notes.

**Comment by ceorourke** [comment]
beep boop commenting for the bot
